Composer/packages/adaptive-form/src/components/fields/OneOfField/utils.ts

```diff
@@ -25,6 +25,18 @@ function getOptionLabel(schema: JSONSchema7): string {
   return type || 'unknown';
 }
 
+const typePriorityWeights = {
```

Comment:
It would be cool to weight the types based on their order in the sdk schema.  Then the schema has control and maybe uischema could provide weight/ordering overrides.

Reply:
@GeoffCoxMSFT Technically speaking it was taking the order of the schema before this PR. So changing the order there would have solved the issue. This PR is just a way to fix the issue client side, which is quicker.